<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageTrucks</class>
 <widget class="QMainWindow" name="ManageTrucks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Trucks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="truckList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="truckID">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Truck ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="truckCapacity">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Truck Capacity (cubic meters)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToList">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To List</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>401</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTruckID">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTruckCapacity">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Locations</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="locationList"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>All items</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="itemList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
